Sources/StreamChatUI/CommonViews/Suggestions/ChatSuggestionsVC.swift

```diff
@@ -189,6 +189,9 @@ open class ChatMessageComposerSuggestionsCommandDataSource: NSObject, UICollecti
 open class ChatMessageComposerSuggestionsMentionDataSource: NSObject,
     UICollectionViewDataSource,
     ChatUserSearchControllerDelegate {
+    /// internal cache for users
+    private(set) var usersCache: [ChatUser]
+    
```

Comment:
I was expecting this to be part of `ChatUserSearchController`. I can see that there is a comment on `ChatUserSearchController` saying: **Note: Currently, no local data will be searched, only remote data will be queried.**

Which means that the goal is to add support for it there. And it makes sense, since the search could be implemented outside of the composer suggestions, so it feels like this should not be done in the composer, but in the search implementation. But not sure how that could be done inside the user search controller, maybe @b-onc has a better understanding of this?